### DISCUSSION TITLE
service name for postgres or mongo container

### DISCUSSION THREAD

**Discussion by VladimirTitov4** [jaegertracing/jaeger#3619] [Q&A] [answered]
Hello everyone. 
I have a SpringBoot application with several microservices. 
I set service name for each service with ```opentracing.jaeger.service-name=gateway``` and this works well. 

But I also have postgres & mongo data base running on docker container. Is it possible see how database handles traces ? And give specific service name to a docker container ?

**Comment by yurishkuro** [comment, marked as answer]
You need to check what kind of support for tracing exists in those databases (I suspect none).
